src/forms/__tests__/RichTextEditor.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { mount } from "enzyme";
+import React from "react";
+import RichTextEditor from "../RichTextEditor";
+
+describe("<RichTextEditor />", () => {
+    it("finds quill text editor", () => {
+        const wrapper = mount(<RichTextEditor />);
+        console.log(wrapper.debug());
```

Comment:
This console.log should be deleted, otherwise it will be logged out with every test